Lib/enum.py

```diff
@@ -414,6 +484,40 @@ def __new__(metacls, cls, bases, classdict, **kwds):
             if _order_ != enum_class._member_names_:
                 raise TypeError('member order does not match _order_')
         #
+        # remove Flag structures if final class is not a Flag
+        if (
+                Flag is None and cls != 'Flag'
+                or Flag is not None and not issubclass(enum_class, Flag)
+            ):
+            delattr(enum_class, '_boundary_')
+            delattr(enum_class, '_flag_mask_')
+            delattr(enum_class, '_all_bits_')
+            delattr(enum_class, '_inverted_')
+        elif Flag is not None and issubclass(enum_class, Flag):
+            # ensure _all_bits_ is correct and there are no missing flags
+            single_bit_total = 0
+            multi_bit_total = 0
+            for flag in enum_class._member_map_.values():
+                if _is_single_bit(flag._value_):
+                    single_bit_total |= flag._value_
+                else:
+                    # multi-bit flags are considered aliases
+                    multi_bit_total |= flag._value_
```

Comment:
it would be better to put `flag._value_` in a local var rather than 3 dereferencing expressions

Reply:
Done.

Reply:
did you forget to push some pending commits?

Reply:
Not forgot, just not gotten to it.  Hope to get it done and committed today (Monday).